### DISCUSSION TITLE
Is UDM Pro Max supported?

### DISCUSSION THREAD

**Discussion by pshirshov** [unifi-utilities/unifi-common#633] [Q&A] [answered]
I'm wondering if UDM Pro Max is supported or not. I would like to upgrade but I depend on this tool and can't find any mentions of the new model.

Thanks!

**Comment by IndyHendrickx** [comment]
I just tested. Looks like it is not supported. I hope they add support soon!

**Comment by bslatyer** [comment, marked as answer]
Fixed by #606
